src/libraries/MathLib.sol

```diff
@@ -2,11 +2,35 @@
 pragma solidity >=0.8.0;
 
 library MathLib {
+    function max(uint256[] memory values) internal pure returns (uint256) {
+        uint256 maxValue = values[0];
+        for (uint256 i = 1; i < values.length; i++) {
```

Comment:
in general I use the i++ form for loop but for libs I would use the optimized way which is

for (uint256 i = 1; i < values.length; ) {

   unchecked {
       ++i;
    }
}

This is ugly but it's a reusable lib and uses a bit less gas